From 770902a0a29fb4ac62358f7eacb3a83d0d9af1c7 Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 25 Nov 2015 10:50:00 +0000
Subject: [PATCH] mdrestore: do not do dynamic linking of libtool libraries

As explained in commit ece49daeff1a3cad765e106d678c608925c9d768, use
-static-libtool-libs instead of -static to allow fallback to the dynamic
linking for libuuid only. Otherwise the build will fail like this:

ld: attempted static link of dynamic object `/usr/lib/libuuid.so'

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
[Sent upstream: http://oss.sgi.com/pipermail/xfs/2015-November/045136.html]
---
 mdrestore/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mdrestore/Makefile b/mdrestore/Makefile
index 5171306..1b34a0e 100644
--- a/mdrestore/Makefile
+++ b/mdrestore/Makefile
@@ -10,7 +10,7 @@ CFILES = xfs_mdrestore.c
 
 LLDLIBS = $(LIBXFS) $(LIBRT) $(LIBPTHREAD) $(LIBUUID)
 LTDEPENDENCIES = $(LIBXFS)
-LLDFLAGS = -static
+LLDFLAGS = -static-libtool-libs
 
 default: depend $(LTCOMMAND)
 
-- 
2.7.4

